packages/apollo/src/components/UpdateChannelMspModal/UpdateChannelMspModal.js

```diff
@@ -331,6 +343,22 @@ class UpdateChannelMspModal extends React.Component {
 				default: 'select_msp_id',
 			},
 		];
+		if (this.props.submit_identity_options && !this.props.isOrdererMSP) {
+			let mspAdmins = this.populateMSPAdmins();
+			fields.push({
+				label: 'Known channel admins',
```

Comment:
this line should really be the name of the translation key to use and not the literal string here, so replace `'Known channel admins'` with `"msp_admin_type_label"` and add that key and this string to the `messages.json` file